process/transaction/shardProcess.go

```diff
@@ -51,29 +51,33 @@ type txProcessor struct {
 	relayedTxV2EnableEpoch         uint32
 	penalizedTooMuchGasEnableEpoch uint32
 	metaProtectionEnableEpoch      uint32
+
+	flagAddFailedRelayedToInvalid         atomic.Flag
+	addFailedRelayedToInvalidDisableEpoch uint32
```

Comment:
addFailedRelayed**Tx**ToInvalid**MBs**DisableEpoch

Reply:
That is so long D. Would leave the name as it is.

Reply:
I have already used the suggestion done by Sebi in my branch

Reply:
will even-out this name